src/compiler/data.ts

```diff
@@ -18,7 +19,7 @@ import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../type';
  *                 If the encoding contains aggregate value, this will also create
  *                 aggregate table as well.
  */
-export default function(model: Model) {
+export function def(model: Model) {
```

Comment:
Fair enough. Do you think it is sensible to say that we only use default if 1) there is only one function exported from a module 2) that function pretty much resembles what you would expect the module to do.